cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -6517,13 +6517,6 @@ search = index=* sourcetype!="WinRegistry" NOT (sourcetype="WinEventLog" user_ty
 | stats latest(user_type) as user_type values(index) as index values(sourcetype) as sourcetype latest(_indextime) as indextime by user vendor_product \
 | makemv delim="\
 " user \
-| append \
```

Comment:
Why we have removed this append??

Reply:
Previously, we were appending sophos users via custom command.

but now we are collecting data from sophos_endpoint source hence those logs will be covered in main search and for that I've written required extraction in props.conf